dataland-frontend/tests/e2e/specs/sfdr/DataUpload.ts

```diff
@@ -0,0 +1,106 @@
+import { describeIf } from "@e2e/support/TestUtility";
+import { admin_name, admin_pw, getBaseUrl } from "@e2e/utils/Cypress";
+import { DataTypeEnum, SfdrData } from "@clients/backend";
+import { getKeycloakToken } from "@e2e/utils/Auth";
+import { generateDummyCompanyInformation, uploadCompanyViaApi } from "@e2e/utils/CompanyUpload";
+import { addsReportsToSfdrDataViaForm, uploadOneSfdrDataset } from "@e2e/utils/SfdrUpload";
+import { FixtureData, getPreparedFixture } from "@sharedUtils/Fixtures";
+import { submitButton } from "@sharedUtils/components/SubmitButton";
+import { toggleRowGroup } from "@sharedUtils/components/ToggleRowFunction";
+
+let companiesWithSfdrData: Array<FixtureData<SfdrData>>;
+let testSfdrCompany: FixtureData<SfdrData>;
+before(function () {
+  cy.fixture("CompanyInformationWithSfdrPreparedFixtures").then(function (jsonContent) {
+    companiesWithSfdrData = jsonContent as Array<FixtureData<SfdrData>>;
```

Comment:
Why setting this variable via a "let" up in the file, if it is only used here?
I suggest:
`const sfdrPreparedFixtures = jsonContent as Array<FixtureData<SfdrData>>;
testSfdrCompany = getPreparedFixture("CompanyInformationWithSfdrData", sfdrPreparedFixtures );`
and removing line 11

Reply:
changed

Reply:
👍